include/bluetooth/scan.h

```diff
@@ -515,6 +515,37 @@ void bt_scan_filter_remove_all(void);
  */
 int bt_scan_params_set(struct bt_le_scan_param *scan_param);
 
+/**@brief Clean connection attempts filter.
+ *
+ * @details Use this function to remove all existing device
```

Comment:
I think `existing device` may be not the best choice. It's hard to tell what `existing` means here?

Maybe use `to remove all entries` instead?

Reply:
Yes good idea. Done